hack/agent-image/test-e2e-dummy-v3.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Test E2E Dummy V3
```

Comment:
what does the v3 refer to?

Reply:
V3 of the image, may be we can just call it test-e2e-another-dummy.service